api/controllers/ConfigController.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * The ConfigController handles the creation and management of SMS app
+ * configuration.
+ */
+
+'use strict';
+
+var ObjectID = require('sails-mongo/node_modules/mongodb').ObjectID;
+
+module.exports = {
+
+  /**
+   * Display campaigns that have configurations set.
+   */
+  list: function(req, res) {
+    YesNoConfig.find({}, function(err, results) {
+      if (err) {
+        return res.status(500).send(err);
+      }
+
+      return res.view('configList', {configs: results});
+    });
+  },
+
+  /**
+   * Display view to create a new campaign configuration.
+   */
+  createView: function(req, res) {
+    return res.view(
+      'configCampaign',
+      {
+        id: 'new',
+        config: null
+      }
+    );
+  },
+
+  /**
+   * Handle the submission request to create a new campaign configuration.
+   */
+  createSubmit: function(req, res) {
```

Comment:
Hmm, it looks like this [Blueprints](http://sailsjs.org/documentation/concepts/blueprints) feature could be helpful for this! :boat: